src/styles/stars.scss

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  &:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
Verify that the image path `../images/star-active.svg` is correct and that the image exists in the specified location. This is important to ensure the active star icons display correctly.